ksorted.py

```diff
@@ -0,0 +1,17 @@
+#Difficulty: how to write a custom comparator for priority queue in python?
```

Comment:
Might I suggest using the heapq library instead.
heapq implements all heap functionality and supports the use of magic functions as well.